build/tasks/esbuild.js

```diff
@@ -23,7 +24,9 @@ module.exports = function(grunt) {
 						outfile: path.join(dest, name),
 						minify: false,
 						bundle: true
-					}).catch(() => process.exit(1));
+					})
+						.then(done)
+						.catch(() => process.exit(1));
```

Comment:
[Based on the Grunt docs](https://gruntjs.com/creating-tasks#why-doesn-t-my-asynchronous-task-complete), it looks like we should pass `false` to `done` here:

```suggestion
						.catch(() => done(false));
```

Don't know how much this matters tho, as you pointed out ESBuild itself emits a "nice" error message prior to exiting the build.

Reply:
Yeah, better to just let Grunt do the work.